Tools/serial/serial_params.c.jinja

```diff
@@ -55,9 +55,8 @@ PARAM_DEFINE_INT32(SER_{{ serial_device.tag }}_BAUD, {{ serial_device.default_ba
  * {{ command.description_extended | replace("\n", " ") }}
  *
  * @value 0 Disabled
-{% for serial_device in serial_devices -%}
- * @value {{ serial_device.index }} {{ serial_device.label }}
-{% endfor -%}
+{% for serial_device in serial_devices %}
+ * @value {{ serial_device.index }} {{ serial_device.label }}{% endfor %}
```

Comment:
Please rather use something like `{%- endfor -%}`, so that we can keep the `endfor` on a new line, which is more readable. You need to check what works.

Reply:
@bkueng 
hi,
sorry for the delay, i was away for a few days.
just pushed the new commit and squashed it with the previous one.